### ISSUE TITLE
Save and import button on Project page reacts to SOW upload

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#1715]
As an analyst
I want the Save button to be reactive to the SOW upload
So that I am not confused by  Save and Import means when I don't do a SOW upload

**Description:**
The save and import button should only visible when a SOW was uploaded, in all other instances it should just be a Save button

# Acceptance Criteria:
- [ ] Save and Import button is visible when a SOW is uploaded by the user
- [ ] In all cases where a SOW was not uploaded, the Save and Import buttons should just be a Save button

**Comment by elliottSlee** [comment]
@MarsRomer the SoW Excel file upload is a required field so the user cannot press save without uploading the file which means they can't save the form without uploading the file. So this card would mislead them to thinking that they can save the form without uploading & importing that file.

**Comment by MarsRomer** [comment]
@elliottSlee  good point, lets talk about it at sprint planning, I would like to re-consider why it's a required field

**Comment by MarsRomer** [comment]
This work was completed in another ticket so I'm closing this one
